charts/bitwardenrs/1.1.1/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: bitwardenrs
+description: Unofficial Bitwarden compatible server written in Rust
+type: application
+version: 1.1.1
+appVersion: 1.18.0
+keywords:
+  - bitwarden
+  - bitwardenrs
+  - bitwarden_rs
+  - password
+  - rust
+home: https://github.com/drlike/truecharts/tree/master/charts/bitwardenrs
+sources:
+  - https://github.com/k8s-at-home/charts/tree/master/charts/bitwardenrs
+  - https://github.com/dani-garcia/bitwarden_rs
+maintainers:
+  - name: DirtyCajunRice
+    email: nick@cajun.pro
```

Comment:
We prefer listing truecharts here, because "DirtyCajunRice" is not working for this project which might result in him getting support calls for things he did not write .